### PR TITLE
Use Admin flag for admins

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

**Comment by dmorrisSL** [comment]
:ship:

File: TrelloMigrate/Transform.fs [hunk 1]
```diff
@@ -28,17 +28,18 @@ module private Profile =
...
           Forename = names.Forename
           Surname = names.Surname
           Rating = 1 
           ImageUrl = imageUrl
-          Bio = String.Empty }
+          Bio = String.Empty 
+          IsAdmin = isAdmin }
 
-    let fromMember (basicMember : BasicMember) = parseFullName basicMember.FullName |> create (getImageUrl basicMember.AvatarHash)
+    let memberToAdminProfile (basicMember : BasicMember) = parseFullName basicMember.FullName |> create (getImageUrl basicMember.AvatarHash) true
```
Comment (pallenSL) [inline TrelloMigrate/Transform.fs]: You could almost call this "createAdmin"

File: TrelloMigrate/Transform.fs [hunk 2]
```diff
@@ -28,17 +28,18 @@ module private Profile =
...
 
-    let fromNameString (fullName : string) = parseFullName fullName |> create defaultImageUrl
+    let nameStringToSpeakerProfile (fullName : string) = parseFullName fullName |> create defaultImageUrl false
```
Comment (pallenSL) [inline TrelloMigrate/Transform.fs]: and this "createSpeaker"

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**